openwisp_utils/admin_theme/admin.py

```diff
@@ -20,3 +22,22 @@ def openwisp_admin(site_url=None):
                 'OPENWISP_ADMIN_INDEX_TITLE',
                 'Network administration')
     )
+
+
+class OpenwispAdminSite(admin.AdminSite):
+    # <title>
+    site_title = getattr(settings,
+                         'OPENWISP_ADMIN_SITE_TITLE',
+                         'OpenWISP Admin')
+    # h1 text
+    site_header = getattr(settings,
+                          'OPENWISP_ADMIN_SITE_HEADER',
+                          'OpenWISP')
+    # text at the top of the admin index page
+    index_title = ugettext_lazy(
+        getattr(settings,
+                'OPENWISP_ADMIN_INDEX_TITLE',
+                'Network administration')
+    )
+
+openwisp_admin_site = import_string(apps.get_app_config('admin').default_site)()
```

Comment:
Move this to a separate file called `site.py` and do something like:

```python
from . import settings as app_settings

admin_site_class = import_string(app_settings.OPENWISP_ADMIN_SITE_CLASS)
admin_site = admin_site_class()
```

This way we avoid possible circular import issues.